src/test/java/com/fasterxml/jackson/core/TestVersions.java

```diff
@@ -50,6 +50,6 @@ private IOContext getIOContext() {
         return new IOContext(StreamReadConstraints.defaults(),
```

Comment:
I think I'd like having `createTestContext()` method in `IOContext` that uses defaults for StreamRead/WriteConstraints, ErrorReportConfiguration(), `ContentReference.unknown()`.

I think I should probably add it concurrently and merge here; not just for this PR but more generally testing.